modules/seqexec/server/src/main/scala/seqexec/server/SequenceGen.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server
+
+
+import cats.Eq
+import cats.implicits._
+import gem.Observation
+import seqexec.model.StepConfig
+import seqexec.model.enum.{Instrument, Resource}
+import seqexec.engine.{Step => EngineStep}
+
+final case class SequenceGen(id: Observation.Id, title: String, instrument: Instrument, steps: List[SequenceGen.Step]) {
```

Comment:
What's the role of this class?

Reply:
That class keeps the "recipe" to produce an `engine.Sequence`, which allows for recreating those sequences when any parameter changes.